src/components/MultipleAvatars.js

```diff
@@ -16,62 +17,59 @@ const defaultProps = {
     optionIsFocused: false,
 };
 
-const MultipleAvatars = props => (
-    <>
-        {props.avatarImageURLs !== undefined && (
-            <>
+const MultipleAvatars = ({avatarImageURLs, optionIsFocused}) => {
+    if (!avatarImageURLs?.length) {
```

Comment:
Just a heads up, we are not using any optional chaining as per our style guide so it would be preferable to avoid it if we can.